### PR TITLE
Remove HTML tags in the reason of a block

### LINKED ISSUE #9271: Remove HTML tags in the `reason` of a block (new "blocked" pages)
Follow up for #9216 

### Describe the problem and steps to reproduce it:
1. Open the following block page: https://addons-dev.allizom.org/en-US/firefox/addon-blocked/firefoxaddon@youtubeenhancer.com/
2. Observe the reason text from "Why was it blocked" section 

### What happened?
HTML present in the reason entered in blocklist admin is not formatted 

### What did you expect to happen?
We should show formatted HTML for the reason field

image

---

(Update) **For QA:** so the idea would be to remove HTML tags in the `reason` of a block (we don't want to support HTML, it should be plain text).

### PR DESCRIPTION
**Pull request by Albaalah**
Fixes #9271

This patch removes HTML tags in _reason_.

### CODE REVIEW

File: src/amo/pages/Block/index.js [hunk 1]
```diff
@@ -71,8 +71,10 @@ export class BlockBase extends React.Component<InternalProps> {
       return null;
     }
 
+    const { __html = '' } = sanitizeHTML(block.reason);
+
     return (
-      <p className="Block-reason">{block ? block.reason : <LoadingText />}</p>
+      <p className="Block-reason">{block ? __html : <LoadingText />}</p>
```
Comment (willdurand) [inline src/amo/pages/Block/index.js]: I think you can inline the call to `sanitizeHTML` here:

```suggestion
      <p className="Block-reason">{block ? sanitizeHTML(block.reason).__html : <LoadingText />}</p>
```

Let me know what you think.

Comment (Albaalah) [reply src/amo/pages/Block/index.js]: Thanks. I made another commit to make the call inline.

**Comment by Albaalah** [comment]
> Great start, thanks. Could you please write a test case too? (there is a file named `TestBlock.js`)

sure, working on it

**Comment by Albaalah** [comment]
@willdurand I have completed the requested change and added a test as well. Kindly review.

File: tests/unit/amo/pages/TestBlock.js [hunk 2]
```diff
@@ -213,6 +213,21 @@ describe(__filename, () => {
     expect(root.find('.Block-reason')).toHaveLength(0);
   });
 
+  it('render reason for a block with HTML tags removed ', () => {
```
Comment (willdurand) [inline tests/unit/amo/pages/TestBlock.js]:
```suggestion
  it('renders the reason with HTML tags removed', () => {
```

Comment (Albaalah) [reply tests/unit/amo/pages/TestBlock.js]: Thank you. I committed the suggestion

**Comment by Albaalah** [comment]
> Great, thanks! I rebased your patch and also removed an assertion that wasn't needed. Good job.

Thank you!

**Comment by willdurand** [comment]
Build passes: https://travis-ci.org/github/mozilla/addons-frontend/builds/667832403

**Comment by willdurand** [comment]
Thank you for your first contribution @Albaalah! /cc @caitmuenster

**Pull request merged**